thegetty/quire
# Accordion shortcode

## Changes
- Include accordion paired shortcode
- Include accordion global controls  shortcode

## Fixes
- Overwrites footnotes plugin `markdown-it` rules to render all footnotes including those within shortcodes using the `markdownify` filter.

## To test
- Make sure you are running the latest version of quire!
- Checkout the `quire` branch `git checkout feature/DEV-7189_accordions`
- Checkout the `quire-starter-default` branch `git checkout feature/DEV-7189_accordion-styles`
- create a test project by running `quire new accordion-test-project ~/Path/to/your/local/quire-starter-default --quire-path ~/Path/to/your/local/quire/packages/11ty`

## Example usage:
Parameters are: `<heading text>, <id>, <open>`
```
{% section "## I am an accordion header" "heading-5" "open" %}
I am accordion content!
{% endsection %}
```

## Example usage of global controls:
To include the global controls, the `accordionGlobalControls` shortcode can be used in a markdown template:
```
{% accordionGlobalControls %}
```
